ivre/db/sql/tables.py

```diff
@@ -79,10 +79,18 @@ def access(d, k):
 
     dbapi_connection.create_function('ACCESS', 2, access)
 
-    def access_astext(d, k):
-        return str(json.loads(d).get(k))
-
-    dbapi_connection.create_function('ACCESS_TXT', 2, access_astext)
+    # FIXME: astext as a manual string cast for JSON.
+    # astext is dirty here because strings value in JSON are not
+    # properly converted to themselves. This may be due to improper
+    # type handling with the custom SQLJSONB.
+    # Proper way to do would be : `cast(x, String)`
+    def astext(x):
+        r = str(x)
```

Comment:
Wouldn't this work:
```Python
        return str(json.loads(r))
```

Reply:
Yes, it would. At first, this line ```r = str(x)``` is useless because argument is a json field so a string already.

The why of ```r.startswith('"')``` is for consistency with simple and double quote.
```>>> str(json.loads('{"1": 1}'))
"{'1': 1}"
>>> str('{"1": 1}')
'{"1": 1}'
```
The last one is still a json, even if astext does not guarantee that output format.